internal/creds_test.go

```diff
@@ -212,8 +212,8 @@ func TestQuotaProjectFromCreds(t *testing.T) {
 	if err != nil {
 		t.Fatalf("got %v, wanted no error", err)
 	}
-	if want, got := "", QuotaProjectFromCreds(cred); want != got {
-		t.Errorf("QuotaProjectFromCreds(validServiceAccountJSON): want %q, got %q", want, got)
+	if want, got := "", GetQuotaProject(cred, ""); want != got {
+		t.Errorf("QuotaProjectFromCreds(validServiceAccountJSON, \"\"): want %q, got %q", want, got)
```

Comment:
Update test function name and error messages from `QuotaProjectFromCreds` to `GetQuotaProject`.